actionpack/lib/system_test_case.rb

```diff
@@ -0,0 +1,57 @@
+require "system_testing/test_helper"
+require "system_testing/driver_adapter"
+
+module Rails
```

Comment:
I actually don't see any dependencies for ActionPack in this patch, so I think we could consider making it a complete stand-alone gem.